CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+By participating to this project, you agree to abide our [code of
+conduct](/CODE_OF_CONDUCT.md).
+
+## Setup your machine
+
+`releaser` is written in [Go](https://golang.org/).
+
+Prerequisites are:
+
+* Build:
+  * `make`
+  * [Go 1.6+](http://golang.org/doc/install)
```

Comment:
Not sure we are using already 1.7+ features (I ran into https://blog.golang.org/subtests recently and using them already everywhere..). I can update accordingly